Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -259,7 +439,19 @@ private void OnEnable()
 
         private void OnDisable()
         {
-            if (IsGlobal)
+            if (IsGlobal || RequiresFocus)
+            {
+                InputSystem.Unregister(gameObject);
+            }
+        }
+
+        private void RegisterGlobalListeners(bool register)
+        {
+            if (register)
+            {
+                InputSystem.Register(gameObject);
```

Comment:
Note: this will need to be `RegisterHandler<IMixedRealitySpeechHandler>`. Same with Unregister.

Reply:
This should be updated to target the correct input handler types.